pandas/core/series.py

```diff
@@ -4067,7 +4067,8 @@ def apply(
             Python function or NumPy ufunc to apply.
         convert_dtype : bool, default True
             Try to find better dtype for elementwise function results. If
-            False, leave as dtype=object.
+            False, leave as dtype=object. Note that conversion does not
+            happen for extension array dtypes, such as Categorical.
```

Comment:
The phrase "Note that conversion does not happen" seems ambiguous to me. Is it that the conversion to object doesn't happen, or the conversion to find a better dtype? Maybe something like "Note that the dtype is always preserved for extension array dtypes, such as Categorical."

Reply:
+1

Reply:
Hi @sukriti1 - do you want to update as per @rhshadrach 's suggestion? Sorry for having led you down the wrong route initially